src/Particle/ParticleAttrib.hpp

```diff
@@ -186,14 +187,15 @@ namespace ippl {
             "ParticleAttrib::gather", policy_type(0, *(this->localNum_mp)),
             KOKKOS_CLASS_LAMBDA(const size_t idx) {
                 // find nearest grid point
-                vector_type l                        = (pp(idx) - origin) * invdx + 0.5;
+                vector_type l                        = (pp(idx) - origin) * invdx + (1.0 - offset);
```

Comment:
is this solution okay with everyone? @srikrrish @matt-frey

Reply:
It seems to me the offset is always used with the default value of 0.5 and hence boils down to the old implementation. Because the non-class function gather didn't pass the argument offset and I don't see anywhere else in the changed files the gather function from ParticleAttrib class directly called. So it seems to me that it was once introduced but forgot to remove? Or did I overlook somewhere?